test/cookie-specs.js

```diff
@@ -0,0 +1,74 @@
+// transpile:mocha
+
+import Cookie from '../lib/cookie.js';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.expect();
+let expect = chai.expect;
+
+describe ('cookies.js', () => { 
+  it ("should properly read a cookie and return the cookie's value given the key", () => {
```

Comment:
Yeah, I put the double quotes because there was a contraction but I should probably just take it out.